chef/cookbooks/neutron/attributes/default.rb

```diff
@@ -20,8 +20,7 @@
 default[:neutron][:debug] = false
 default[:neutron][:verbose] = false
 default[:neutron][:dhcp_domain] = "openstack.local"
-default[:neutron][:networking_mode] = "gre"
-default[:neutron][:networking_plugin] = "openvswitch"
+default[:neutron][:networking_plugin] = "ml2"
 default[:neutron][:cisco_support] = false
```

Comment:
Not related to your PR, but can you drop this attribute? I don't think it's used.